### PR TITLE
Models: added enum for weekly goals and adjusted rest of the model

### PR DESCRIPTION
**Pull request by skorpiom**
update

### CODE REVIEW

File: Models/WeeklyGoals.model.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export enum WeeklyGoals {
```
Comment (robert-czwartosz) [inline Models/WeeklyGoals.model.ts:1]: enum name WeeklyGoal wouldn't be better?

**Review by robert-czwartosz — COMMENTED** [review]
I suggest to use singular for naming enum.
Component profileInfo needs information about current weight and current weekly goal.
Could you add these two properties to interface User?

**Comment by skorpiom** [comment]
You're right component profileInfo needs that data - but logic function responsible for passing informations to that component gonna get user object and retrive currentWeight from 'weights' array and weeklyGoal from array 'Goals' - in both cases it'll be a last object in the array. 
If we wanna duplicate that date (place copy in User.model.ts) we'd have to create some additional logic function to update that things in certain point. 

Let me know what you think ;)

File: Models/WeeklyGoals.model.ts (hunk 1 above)
Comment (skorpiom) [reply Models/WeeklyGoals.model.ts:1]: Sure, it'll be changed.

**Comment by robert-czwartosz** [comment]
It's good idea.
Are you sure that both arrays *always* aren't empty?

**Comment by skorpiom** [comment]
Theoretically in each array should be at least one object because during the account creation phase user must input currentWeight and select weekly goal.

**Review by fszwedo — APPROVED** [review]

**Comment by robert-czwartosz** [comment]
OK. I will change my login button unit implementation, according this idea.

**Review by robert-czwartosz — APPROVED** [review]

**Pull request merged**